cob_bringup/package.xml

```diff
@@ -66,12 +62,8 @@
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>costmap_2d</exec_depend>
```

Comment:
then I think that we can remove the dependencies, cob_bringup depends on cob_hardware_config, and cob_hardware_config should define the dependencies.